src/test/java/org/apache/drillui/test/framework/testng/secure/login/LoginTest.java

```diff
@@ -16,23 +16,90 @@
  */
 package org.apache.drillui.test.framework.testng.secure.login;
 
+import org.apache.drillui.test.framework.initial.PropertiesConst;
 import org.apache.drillui.test.framework.initial.TestProperties;
+import org.apache.drillui.test.framework.initial.WebBrowser;
+import org.apache.drillui.test.framework.pages.BasePage;
+import org.apache.drillui.test.framework.pages.QueryResultsPage;
+import org.apache.drillui.test.framework.steps.webui.*;
 import org.apache.drillui.test.framework.testng.secure.BaseSecureTest;
+import org.junit.Assert;
```

Comment:
Please, change JUnit to the TestNG framework here.

Reply:
Done